### ISSUE TITLE
[Nodecore v0.2.14] Blockchain got stuck

### ISSUE THREAD

**Issue by a123b** [VeriBlock/nodecore-releases#103]
My blockchain got stuck at block 151000. Restarting nodecore does not solve the problem.
Instead of adding the next block, it periodically prints the following message:
```
Sep 05, 2018 10:39:33 AM com.google.common.eventbus.EventBus$LoggingHandler handleException
SEVERE: Exception thrown by subscriber method a(nodecore.d.a.q) on subscriber nodecore.d.k@33a55bd8 when dispatching event: nodecore.d.a.q@4755175c
java.lang.NullPointerException
        at nodecore.core.a.q.d(DefaultBlockchain.java:96)
        at nodecore.d.k.a(PeerEventListener.java:187)
        at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:87)
        at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:72)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
```
As my other node synced past 151000, I assume my nodecore.dat got corrupted somehow and nodecore doesn't know how to fix it.

**Comment by huglester** [comment]
probably this can be closed, since testnet restarted

**Comment by max-sanchez** [comment]
This bug was fixed in ~v0.2.17, closing.
